cou/utils/juju_utils.py

```diff
@@ -99,15 +105,16 @@ async def _async_get_model(model_name=None) -> Model:
     return model
 
 
-async def _disconnect(model: Model):
+# pylint: disable=broad-exception-caught
```

Comment:
We should rather define more specif Exception, instead of using `except Exception`. What do you think?

Reply:
I could not find specific exception  when disconnecting. Thus I used exception.

Reply:
Yes, I could not found it either in source code, but I would expect something like `JujuError` here. We need to testes these properly.